app/models/manageiq/providers/vmware/network_manager/cloud_network/org_vdc_net.rb

```diff
@@ -0,0 +1,3 @@
+class ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::OrgVDCNet <
+    ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork
```

Comment:
If you're splitting this up just to avoid a rubocop line-length warning I think its more readable to leave it on one line.
Opinions?